models/marts/core/int_all_dag_relationships__chained_views.sql

```diff
@@ -0,0 +1,9 @@
+-- creates a cte called all_relationships that will either use "with recursive" or loops depending on the DW
+{{ dbt_project_evaluator.recursive_dag(filter_views=true) }}
+
+select 
+    * 
+from all_relationships
+where 
+    distance <> 0
```

Comment:
Can we move this filter downstream, just so this model is consistent with `int_all_dag_relationships`?